setup/test/integration/conftest.py

```diff
@@ -29,4 +29,4 @@ def pytest_sessionstart(session):
     # conan manually here. Can be removed once conan is installed
     # globally.
     if os.environ["VELOCITAS_TEST_LANGUAGE"] == "cpp":
-        subprocess.check_call(["python", "-m", "pip", "install", "conan==1.60.2"])
+        subprocess.check_call(["python", "-m", "pip", "install", "conan==1.63.0"])
```

Comment:
hardcoded versions in files/test could potentially be hard to troubleshoot. We can keep it because we do it on many places, but we could potentially improve here

Reply:
If we were to put this in requirements.txt then this would be installed even for Python, not really sure where to put it